php/src/Google/Protobuf/Internal/Message.php

```diff
@@ -987,7 +987,7 @@ protected function mergeFromArray(array $array)
                     'Invalid message property: ' . $key);
             }
             $setter = $field->getSetter();
-            if ($field->isWrapperType()) {
+            if ($field->isWrapperType() && !($field->isRepeated() || $field->isMap())) {
```

Comment:
A wrapper type field cannot be map field. Only the value field of a map entry can be a wrapper type message.

Reply:
Ah ok, will remove check for isMap